### PR TITLE
Update api.md

### PR DESCRIPTION
**Pull request by Morpheus-zzy**
Add documentation to use APIs that require admin access

### Description

When using security APIs that require super admin access, like [Distinguished names](https://opensearch.org/docs/latest/security/access-control/api/#distinguished-names), [Certificates](https://opensearch.org/docs/latest/security/access-control/api/#certificates).
The configuration `plugins.security.restapi.admin.enabled: true` is required, otherwise unable to use the relevant APIs.

### Issues Resolved
- 

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by hdhalter** [comment]
Thanks, @Morpheus-zzy ! @scrawfor99 , can you please review for technical accuracy? Thanks.

File: _security/access-control/api.md [hunk 1]
```diff
@@ -29,6 +29,13 @@ plugins.security.restapi.roles_enabled: ["<role>", ...]
 ```
 {% include copy.html %}
 
+For APIs related to `Distinguished names` or `Certificates` that require super admin access, you should enable RESR API admin configuration in `opensearch.yml`:
```
Comment (scrawfor99) [inline _security/access-control/api.md:32]:
```suggestion
For APIs related to `Distinguished names` or `Certificates` that require super admin access, you should enable REST API admin configuration in `opensearch.yml`:
```

**Review by scrawfor99 — APPROVED** [review]
Thank you! Just one typo

File: _security/access-control/api.md [hunk 2]
```diff
@@ -29,6 +29,13 @@ plugins.security.restapi.roles_enabled: ["<role>", ...]
 ```
 {% include copy.html %}
 
+For APIs related to `Distinguished names` or `Certificates` that require super admin access, you should enable REST API admin configuration in `opensearch.yml`:
```
Comment (vagimeli) [inline _security/access-control/api.md:32]:
```suggestion
If you're working with APIs that manage `Distinguished names` or `Certificates` that require super admin access, enable the REST API admin configuration in your `opensearch.yml` file as shown in the following setting example:
```

**Review by vagimeli — APPROVED** [review]

**Pull request merged**